p2p/proto/block.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+import "p2p/proto/common.proto";
+import "google/protobuf/timestamp.proto";
+
+// for now, we assume a small consensus, so this fits in 1M. Else, these will be repeated
+message Signatures {
+    BlockID id = 1;
+    repeated Signature signatures = 2; // can be more explicit here about the signature structure as this is not part of account abstraction
+}
+
+// Note: commitments may change to be for the previous blocks like comet/tendermint
+// hash of block header sent to L1
+message BlockHeader {
```

Comment:
I recommend a `hash` field also since we can't recalculate them for some blocks. Otherwise, we are blocked by it until we figureout what to do with old blocks.

Reply:
Noted